Himeshi test

HimeshiS opened the pull request:
please add

HimeshiS committed:
trying it out

a few things changed

HimeshiS committed:
some more edits

forgot to do this last time

Zeavz commented on README.md:
```diff
@@ -1 +1,2 @@
 # POC_DIN_SCAN
+#there's nothing here 
```
Very interesting change 🤔

HimeshiS replied:
i thought so too.....please approve

Zeavz commented on cordova-swift3-plugin-example-master/src/ios/ModusEchoSwift.swift:
```diff
@@ -499,3 +499,4 @@ class ModusEchoSwift : CDVPlugin, NSObject, AVCaptureVideoDataOutputSampleBuffer
         }
     }
 }
+print("my god what is this?!?!")
```
Too many exclamations!

HimeshiS replied:
oh oh will remove some!

HimeshiS committed:
made the requested changes

thanks! removed additional punctuation because i agree with you, they are unnecessary, silly me haha

Zeavz committed:
Deleting extra indent

Zeavz commented:
I made some changes

HimeshiS committed:
new file
